src/mock/destination.js

```diff
@@ -0,0 +1,29 @@
+import {getRandomInteger, getRandomArrayElement} from '../utils.js';
+import {DESCRIPTIONS, NAMES, PICTURE_SRC} from '../const.js';
+import {TASK_COUNT} from '../const.js';
+
+const generateDescription = () => {
+  const carentDescription = [];
+  for (let i = 1; i <= getRandomInteger(); i++) {
+    carentDescription.push(getRandomArrayElement(DESCRIPTIONS));
+  }
```

Comment:
косяк, getRandomInteger - срабатывает на каждой итерации цикла

```suggestion
const count = getRandomInteger();
  for (let i = 1; i <= count; i++) {
    carentDescription.push(getRandomArrayElement(DESCRIPTIONS));
  }
```